packages/ai-jsx/src/batteries/sidekick/platform/system-message.tsx

```diff
@@ -29,21 +29,20 @@ export function SidekickSystemMessage({
   const dateTimeSystemMessage = (
     <SystemMessage>
       The current date and time is: {new Date().toLocaleString()}. The current day of the week is: {currentDate}. The
-      user's local time zone is {timeZone}, which has an offset from UTC of {timeZoneOffset}. When giving dates, use the
-      user's time zone. If the user does not specify a date or time, assume it is intended either now or in the near
-      future.
+      user's local time zone is {timeZone}. When giving dates, use the user's time zone. If the user does not specify a
+      date or time, assume it is intended either now or in the near future.
     </SystemMessage>
   );
 
   const responseFramingSystemMessage = (
     <SystemMessage>
-      Do not say "according to the documents I have been given"; just answer as if you know the answer. When you see
-      timestamps in your source data, format them in a nice, human-readable way. Speak politely but casually. Say `use`
+      Do not say "according to the documents I have been given", just answer as if you know the answer. When you see
+      timestamps in your source data, format them in a human-readable way. Speak politely but casually. Say `use`
       instead of `utilize`.
     </SystemMessage>
   );
 
-  const baseComponentNames = ['Card', 'Citation'];
+  const baseComponentNames = ['Citation'];
```

Comment:
If we're taking out Card, we should take it out everywhere, including from the MDX examples.

Reply:
I was hoping this was the simplest way to remove it, since I don't want it being rendered but might want to bring back a modified form later

Reply:
Given that everything in the system message competes for attention, I think we should go through the effort to remove it universally. We can always bring it back via the git history.